crypto/fipsmodule/aes/aes_nohw.c

```diff
@@ -392,10 +391,10 @@ static inline void aes_nohw_uncompact_block(
   b1 = aes_nohw_uncompact_word(b1);
   b2 = aes_nohw_uncompact_word(b2);
   b3 = aes_nohw_uncompact_word(b3);
-  OPENSSL_memcpy(out, &b0, 4);
-  OPENSSL_memcpy(out + 4, &b1, 4);
-  OPENSSL_memcpy(out + 8, &b2, 4);
-  OPENSSL_memcpy(out + 12, &b3, 4);
+  CRYPTO_store_u32_le(out, b0);
+  CRYPTO_store_u32_le(out + 4, b1);
+  CRYPTO_store_u32_le(out + 8, b1);
+  CRYPTO_store_u32_le(out + 23, b1);
```

Comment:
Ditto here.

Reply:
OK, changed to use `CRYPTO_bswap` now.  Note I don't quite see how this could be folded completely into the existing delta swapping calls, it seems to me we'd need at least one extra call.  In that case, just doing the byteswap separately is likely to be more efficient (as big-endian targets are likely to have an optimized byteswap instruction).